Jenkinsfile

```diff
@@ -279,6 +279,7 @@ def doBuildWindows(String buildType, boolean isUWP, String platform) {
             }
             if(!isUWP) {
                 def environment = environment() << "TMP=${env.WORKSPACE}\\temp"
+                environment << 'UNITTEST_PROGRESS=1'
```

Comment:
What is this all about?

Reply:
Windows was aborting its test run and I used this to track down which test was failing. The logs on CI were a bit cryptic about what was going wrong and I think being verbose could save us time in the future. We monitor progress in non-windows tests as well but it was forgotten for this mode. Since it is functionally unrelated to this PR in particular, so I am happy to move it to a separate PR if preferred.

Reply:
Thanks for explaining. I'm fine with keeping it in this PR.